doc/nrf/known_issues.rst

```diff
@@ -791,6 +791,13 @@ The combination of nRF Secure Immutable Bootloader and MCUBoot fails to upgrade
 Build system
 ============
 
+.. rst-class:: v1-5-0 v1-5-1 v1-6-0 v1-6-1
+
+NCSDK-10672: dfu_application.zip is not updated during build (configuration with MCUboot).
+  Because of missing dependencies the dfu_application.zip may not be properly updated after code or configuration changes.
```

Comment:
```suggestion
   In the configuration with MCUboot, the :file:`dfu_application.zip` might not be properly updated after code or configuration changes, because of missing dependencies.